js/timers.js

```diff
@@ -0,0 +1,70 @@
+// A collection of globally accessible timers
+
+const timers = {
+  poison: function() {
+    //poison elements
```

Comment:
I have concerns with this way of listing timers. first thing I can think of is there would be no way to stack two similar timers as they are only properties within a timers object. It makes more sense to me for timers to be an array and when a timer is created, it is added into the array stack. once a timer is finished, it is then disposed of. This way we can have multiple timers of the same type running at once, i.e. two attack up potion timers that stack and add to your  attack per each potion used.

Reply:
If the stats stack, why not just increase the effect of that timer? You drink one atk-up potion, you get +5 atk. Then another atk-up potion is used, now you get +10. Change the SST of the atk-up effect and apply it as the effect.